web/src/elements/ak-dual-select/types.ts

```diff
@@ -2,9 +2,21 @@ import { TemplateResult } from "lit";
 
 import { Pagination } from "@goauthentik/api";
 
-// Key, Label (string or TemplateResult), (optional) string to sort by. If the sort string is
-// missing, it will use the label, which doesn't always work for TemplateResults).
-export type DualSelectPair<T = never> = [string, string | TemplateResult, string?, T?];
+//
+// - key: string
```

Comment:
This is pure "cleaning up the documentation."  Typescript's "named tuples" are lovely for self-documenting code.